war/src/main/less/modules/sidebar-tasks.less

```diff
@@ -0,0 +1,128 @@
+@import '../abstracts/colors.less';
+@import '../abstracts/font.less';
+
+#tasks {
+  margin-bottom: 20px;
+  display: inline-block;
+
+  // TODO: investigate this on the git history to see if removing it will
+  // cause a problem
+  max-width: 325px;
+
+  // The task list is touching the side of the screen. This should be removed
+  // once .side-panel loses it's padding.
+  width: calc(100% + 15px);
+  margin-left: -15px;
+}
+
+#tasks .task {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  position: relative;
+  font-size: @font-size-sm;
+  font-weight: 500;
+}
+.task.disabled {
+  opacity: 0.5;
+  background-color: transparent;
+}
+
+.task-link-wrapper {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: 2px 0;
+}
+
+.task-link,
+.task-disabled-link {
+  display: inline-flex;
+  flex: 1;
+  align-items: center;
+  padding: 0.5rem 0 0.5rem 1.25rem;
+  max-width: 100%;
+}
+.task-link:link,
+.task-link:visited {
+  color: @text-color;
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    background-color: #F8F8F8;
```

Comment:
Shouldn't this be in the colors file?

Reply:
TBH yes, but I have yet to figure out how all greys fill in the colour palette. I'm waiting until I have a better idea of the greyscale before extracting them to variables. I can extract them now if you think it's needed.

Reply:
No problem, extract them when you have the full idea.